web/templates/app.tmpl.html

```diff
@@ -3,7 +3,8 @@
     data-release-version="{{.Data.ReleaseVersion}}"
     data-go-version="{{.Data.GoVersion}}"
     data-controller-namespace="{{.ControllerNamespace}}"
-    data-uuid="{{.UUID}}">
+    data-uuid="{{.UUID}}"
+    data-jaeger="{{.Jaeger}}">
```

Comment:
Is this a `jaegerEnabled` variable?

Reply:
the jaeger url variable? As we need the concrete URl to use when routing. Did I understand your question correctly?

Reply:
The variable name doesn't tell me what it is. This is the `jaegerURI` then?

Reply:
Ah, Right having a `showJaeger`makes sense, as we don't use the url anyway (because we directly proxy), but React is acting weird to pass boolean parameters. It doesn't work, and the internet agrees that its a thing. I could be wrong here! 

I can take another look if its important!

Reply:
I don't actually care if it is a boolean, making it a string of "true" would work. It is more that that variable says what it does =)